azimuth/plots/dataset_warnings.py

```diff
@@ -85,19 +73,17 @@ def nb_samples_plot(
     fig.update_yaxes(ticks="outside", ticklen=24, tickcolor="white")
 
     # Generate warning indicators on the left axis
-    x_warning = -0.037
-    for idx, label in enumerate(order):
-        common_args = dict(
-            xref="paper",
-            showarrow=False,
-            x=x_warning,
-            y=idx,
-            font=dict(color=Colors.Orange, size=AXIS_FONT_SIZE),
-        )
-        if alert_train[label]:
-            fig.add_annotation(text="◒", **common_args)
-        if alert_eval[label]:
-            fig.add_annotation(text="◓", **common_args)
+    alert_text_per_split = {DatasetSplitName.eval: "◓", DatasetSplitName.train: "◒"}
+    common_args = dict(
+        xref="paper",
+        showarrow=False,
+        x=-0.037,
+        font=dict(color=Colors.Orange, size=AXIS_FONT_SIZE),
+    )
+    for split, alert in alert_per_cls_per_split.items():
+        for idx, cls in enumerate(order):
+            if alert[cls]:
+                fig.add_annotation(text=alert_text_per_split[split], y=idx, **common_args)
```

Comment:
Now that `common_args` is used in only one place (good job! 👍), feel free to integrate it directly, or not.
```suggestion
    for split, alert in alert_per_cls_per_split.items():
        for idx, cls in enumerate(order):
            if alert[cls]:
                fig.add_annotation(
                    text=alert_text_per_split[split],
                    xref="paper",
                    showarrow=False,
                    x=-0.037,
                    y=idx,
                    font=dict(color=Colors.Orange, size=AXIS_FONT_SIZE),
                )
```

Reply:
Right, I missed that one! Thanks!!